docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2.4'
+
+services:
+  mongo:
+    image: mongo:4.0.1
+    ports:
+      - '27017'
+    # /host for ad-hoc transfers in/out of container
+    # /data for DB data itself
+    volumes:
+      - './data/host:/host'
+      - './data/mongo:/data'
+
+  hapi:
+    build: .
+    command: node --inspect=[::]:9229 code/main.js
```

Comment:
Glad you're listening to the loopback port here, otherwise you can get debugging issues.  I was able to debug this app with VS Code Node debugger.